### ISSUE TITLE
deep linking with react-native-router-flux v4

### ISSUE THREAD

**Issue by chris-topher19** [aksonov/react-native-router-flux#3421]
-->
- react-native-router-flux v4.0.6
- react v16.4.2
- react-native v0.55.4

### Expected behaviour
I am trying to implement the deep linking in my react-native app and used the **uriPrefix={"https://google.com"}** and **path='/homeTry'** in my scene to some path. On typing some url eg: **"https://google.com/homeTry" in browser** should open my application.

### Actual behaviour
Nothing is happening on typing the url in my emulator's browser.

### Steps to reproduce
1. Run the application using react-native run-android
2. open the emulator brower, type the following link https://google.com/homeTry

Here's my router 
 

> <Router onStateChange={stateHandler} sceneStyle={styles.scene} uriPrefix={'https://google.com'}>
>    <Scene key="home"  path="homeTry"/>
> </Router>

Am I missing something, I have tried everything mentioned in the documentation and also tried running the example provided in this repository. But unable to find success. Help me out.

**Comment by KPS250** [comment]
I went through the documentation and couldn't find any support for deep-linking in react-native-router-flux library.

@chris-topher19 Did you make any progress with deep-linking with router flux?

**Comment by chris-topher19** [comment]
@KPS250 no progress using router flux, Ended up using Linking library mentioned in official document. Works well for me as  of now.

**Comment by KPS250** [comment]
Thanks, @chris-topher19

**Comment by RicardoBrito1938** [comment]
@bhavna-18 i looked but did not find, what is the official Linking library?

**Comment by johnsonsy** [comment]
@RicardoBrito1938 Here you go, https://reactnative.dev/docs/linking

**Comment by bhavna-18** [comment]
@RicardoBrito1938 The one mentioned by the @johnsonsy

**Comment by RicardoBrito1938** [comment]
> @RicardoBrito1938 The one mentioned by the @johnsonsy

Thank you!!

**Comment by yogendrajs** [comment]
> @RicardoBrito1938 Here you go, https://reactnative.dev/docs/linking

Hi thanks, I tried this, but how do I define the path to open a specific screen? Any workaround?

**Comment by RicardoBrito1938** [comment]
> > @RicardoBrito1938 Here you go, https://reactnative.dev/docs/linking
> 
> Hi thanks, I tried this, but how do I define the path to open a specific screen? Any workaround?

Hello, i don't work with router-flux anymore, but at time, i sent with a parameter on URL, then i had an useEffect which executed an "Action" once this param was received from the URL

**Comment by yogendrajs** [comment]
@RicardoBrito1938 thanks, I'll check that. Anyways, did you move to React-navigation?

**Comment by RicardoBrito1938** [comment]
@yogendrajs , yes, i am using integrated with expo, pretty straight forward for deep link
